lib/maps.rb

```diff
@@ -20,8 +20,9 @@ def place_to_geodata_point(segment, storyline_timezone)
      'endtime' => Time.parse(segment['endTime']).to_i}
   end
 
-  def trackpoints_to_geodata_multiline(activity, trackpoints)
+  def move_to_geodata_multiline(move)
     coordinates = []
+    trackpoints = move['trackPoints']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.